logexp/Makefile

```diff
@@ -0,0 +1,34 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT ?= google-containers
```

Comment:
Oops.. Sorry for the blooper, fixed it. I didn't want to push any image to k8s-testimages before we agree upon this. So:
- Would I be able to push to it (given that I have permission to push to google-containers) or would you( /testinfra) do the pushing part?
- Given that this (in part) does the job of cluster/log-dump.sh, maybe this tool should be available under the more general project viz. google-containers?
- We might want to decide upon the tag based on the project(?) Like v0.1.0 or something if we want to use google-containers, or git describe --tags based tag for k8s-testimages?